<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Qscoreboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>30</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>08</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="8" rowspan="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>+1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_12">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10" rowspan="5">
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>+1</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="9" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/cnc500.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="5">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Qscoreboard</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" rowspan="3">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Available Time Out</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_6">
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="9" rowspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>+1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9" rowspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" rowspan="2" colspan="7">
     <widget class="DigitalClock" name="main_clock_widget" native="true"/>
    </item>
    <item row="8" column="9" rowspan="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>+1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Shot Clock</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="10" rowspan="3">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Available Time Out</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_7">
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QPushButton" name="pushButton_15">
      <property name="text">
       <string>Sound</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="3" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="intValue" stdset="0">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QPushButton" name="pushButton_13">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" rowspan="2">
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>+1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="5">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QPushButton" name="pushButton_11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start/Stop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Away</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="9">
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/cwl500.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Main Clock</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QPushButton" name="pushButton_14">
      <property name="text">
       <string>Start/Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" rowspan="3">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Period</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_4">
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QPushButton" name="pushButton_17">
      <property name="text">
       <string>Add Screen 30</string>
      </property>
     </widget>
    </item>
    <item row="12" column="6">
     <widget class="QPushButton" name="pushButton_16">
      <property name="text">
       <string>Add Screen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QWidget</extends>
   <header>digitalclock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
